core/node/crypto/testutil.go

```diff
@@ -99,7 +101,14 @@ func initSimulated(ctx context.Context, numKeys int) ([]*Wallet, *simulated.Back
 		genesisAlloc[wallets[i].Address] = types.Account{Balance: Eth_100}
 	}
 
-	backend := simulated.NewBackend(genesisAlloc, simulated.WithBlockGasLimit(30_000_000))
+	// With very short block period the eth node log indexing service can't keep up indexing logs.
+	// This can cause a deadlock when retrieving logs through the api. Disabled the indexing service,
+	// this hasn't got a noticeable performance impact in tests.
+	disableLogIndexing := func(nodeConf *node.Config, ethConf *ethconfig.Config) {
```

Comment:
Should the PR title be changed to briefly mention another change apart from bumping the package version?